include/gamepad/gamepad.hpp

```diff
@@ -75,7 +111,9 @@ class Gamepad {
         static Gamepad partner;
     private:
         Gamepad(pros::controller_id_e_t id)
-            : controller(id) {}
+            : controller(id) {
+            this->add_screen(defaultScreen);
+        }
```

Comment:
Usually ctors are declared in headers an defined in source files, even if its small like this. This is for consistency's sake